File: src/main/java/de/fhopf/akka/Execution.java
package de.fhopf.akka;

import org.apache.lucene.index.IndexWriter;

/**
 *
 * @author Florian Hopf, http://www.florian-hopf.de
 */
public interface Execution {

    public void downloadAndIndex(String path, IndexWriter writer);
    
}

File: src/main/scala/de/fhopf/akka/Execution.scala
package de.fhopf.akka

import org.apache.lucene.index.IndexWriter

/**
 * @author Florian Hopf, http://www.florian-hopf.de
 * @author Alfredo Serafini, http://www.seralf.it
 */
abstract class Execution {
  def downloadAndIndex(path: String, writer: IndexWriter)
}